libcxx/utils/ci/buildkite-pipeline.yml

```
@@ -274,6 +278,13 @@ steps:
     <<: *android_agent
     <<: *common
 
+- group: ':ohos: OpenHarmony'
```

```suggestion
- group: ' OpenHarmony'
```
This looks better IMO.

---

Fixed